app.rb

```diff
@@ -4,6 +4,10 @@
 # IAM - a resource usage metric collection and reporting system
 class Iam < Sinatra::Base
   set :show_exceptions, :after_handler
+  enable :method_override
+  set :port, 4567
```

Comment:
I would put the port and bind variables in a config file and *default* them to these values.